checkstyle-checks/src/main/resources/sonatype/checkstyle-configuration.xml

```diff
@@ -162,7 +162,7 @@
       <property name="allowThrowsTagsForSubclasses" value="true"/>
     </module>
     <module name="MethodName">
-      <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9_]*$"/>
+      <property name="format" value="^[a-z][a-zA-Z0-9_]*$"/>
```

Comment:
+1, but I'm not sure we should have the `_` in there.
Perhaps a topic for another day.